src/commands/deployContainerApp/deployContainerApp.ts

```diff
@@ -25,6 +26,13 @@ export async function deployContainerApp(context: IActionContext, node?: Contain
     const subscriptionContext: ISubscriptionContext = createSubscriptionContext(item.subscription);
     const subscriptionActionContext: ISubscriptionActionContext = { ...context, ...subscriptionContext };
 
+    if (item.containerApp.revisionsMode === KnownActiveRevisionsMode.Multiple) {
+        throw new Error(localize('multipleRevisionsNotSupported', 'The container app "{0}" cannot be updated using "Deploy to Container App..." while in multiple revisions mode.', item.containerApp.name));
```

Comment:
I think it'd be helpful to say how they can deploy it if not "Deploy to Container App." Same for the error below. I assume that you have to just right-click the specific revision or container depending on the scenario?

Reply:
Yeah good idea.  I eventually would like to enable use of this command on the revisions directly, but I think that's a change for a future release